### ISSUE TITLE
after run the framwork in emulater when start sync stop working

### ISSUE THREAD

**Issue by modekano** [Odoo-mobile/framework#280]
the error is :
11-30 09:10:29.961 19349-19374/com.odoo E/AndroidRuntime: FATAL EXCEPTION: ModernAsyncTask #1
                                                          Process: com.odoo, PID: 19349
                                                          java.lang.RuntimeException: An error occurred while executing doInBackground()
                                                              at android.support.v4.content.ModernAsyncTask$3.done(ModernAsyncTask.java:161)
                                                              at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:355)
                                                              at java.util.concurrent.FutureTask.setException(FutureTask.java:222)
                                                              at java.util.concurrent.FutureTask.run(FutureTask.java:242)
                                                              at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
                                                              at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
                                                              at java.lang.Thread.run(Thread.java:818)
                                                           Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String com.odoo.core.orm.OModel.getModelName()' on a null object reference
                                                              at com.odoo.core.orm.provider.BaseModelProvider.setMatcher(BaseModelProvider.java:85)
                                                              at com.odoo.core.orm.provider.BaseModelProvider.query(BaseModelProvider.java:92)
                                                              at android.content.ContentProvider.query(ContentProvider.java:950)
                                                              at android.content.ContentProvider$Transport.query(ContentProvider.java:210)
                                                              at android.content.ContentResolver.query(ContentResolver.java:478)
                                                              at android.support.v4.content.ContentResolverCompatJellybean.query(ContentResolverCompatJellybean.java:29)
                                                              at android.support.v4.content.ContentResolverCompat$ContentResolverCompatImplJB.query(ContentResolverCompat.java:57)
                                                              at android.support.v4.content.ContentResolverCompat.query(ContentResolverCompat.java:125)
                                                              at android.support.v4.content.CursorLoader.loadInBackground(CursorLoader.java:59)
                                                              at android.support.v4.content.CursorLoader.loadInBackground(CursorLoader.java:37)
                                                              at android.support.v4.content.AsyncTaskLoader.onLoadInBackground(AsyncTaskLoader.java:299)
                                                              at android.support.v4.content.AsyncTaskLoader$LoadTask.doInBackground(AsyncTaskLoader.java:57)
                                                              at android.support.v4.content.AsyncTaskLoader$LoadTask.doInBackground(AsyncTaskLoader.java:45)
                                                              at android.support.v4.content.ModernAsyncTask$2.call(ModernAsyncTask.java:138)
                                                              at java.util.concurrent.FutureTask.run(FutureTask.java:237)
                                                              at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112) 
                                                              at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587) 
                                                              at java.lang.Thread.run(Thread.java:818) 
11-30 09:10:29.980 804-1116/system_process W/ActivityManager:   Force finishing activity com.odoo/.OdooActivity
11-30 09:10:29.991 19349-19376/com.odoo I/Process: Sending signal. PID: 19349 SIG: 9
11-30 09:10:30.005 804-847/system_process W/InputDispatcher: channel '27ad0894 com.odoo/com.odoo.OdooActivity (server)' ~ Consumer closed input channel or an error occurred.  events=0x9
11-30 09:10:30.005 804-847/system_process E/InputDispatcher: channel '27ad0894 com.odoo/com.odoo.OdooActivity (server)' ~ Channel is unrecoverably broken and will be disposed!
11-30 09:10:30.015 749-749/? E/EGL_emulation: tid 749: eglCreateSyncKHR(1181): error 0x3004 (EGL_BAD_ATTRIBUTE)

**Comment by donghd** [comment]
I think it should change as below、because it may need a bit time to collecting data before model available.
com.odoo.core.orm.provider.BaseModelProvider.setMatcher(BaseModelProvider.java:85)
```
setMatcher(uri);
 if (mModel == null)
    return null;
```
↓↓↓
```
 if (mModel == null)
    return null;
setMatcher(uri);
```

**Comment by modekano** [comment]
Thanks  DONGHD 
I solved as issue #276
